### PR TITLE
Fix videopreview component bug

### PR DESCRIPTION
**Pull request by michhyun1**
**Issue #:** 
https://github.com/aws/amazon-chime-sdk-component-library-react/issues/314

**Description of changes:**
This fixes a bug where the VideoPreview component would not stop the media stream on the video preview video element because a ref was being unset before the unmount method could use the value of the ref.

Instead of using the ref in the unmount method - we just save the ref value locally, and then use the variable when we unmount. 

**Testing**
1. Have you successfully run `npm run build:release` locally? 
yes

2. How did you test these changes?
Join a meeting, verify that the light next to the webcam indicating an active media stream is turned off when we join the meeting. Then, turn the camera on while in the meeting, and verify that the light next to the webcam is turned on. Then, leave the meeting and verify that the light is now turned off. This indicates all media streams are not active. 

3. If you made changes to the component library, have you provided corresponding documentation changes?
yes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/components/sdk/PreviewVideo/index.tsx [hunk 1]
```diff
@@ -28,8 +28,8 @@ export const PreviewVideo: React.FC<BaseSdkProps> = (props) => {
     if (!audioVideo || !selectedDevice || !videoEl.current) {
       return;
     }
-
     let mounted = true;
+    let videoElement = videoEl.current;
```
Comment (ltrung) [inline src/components/sdk/PreviewVideo/index.tsx:32]: Should this be const?

**Comment by ltrung** [comment]
You may want to update to Node 16 with NPM 7. It seems you changes package-lock.json to v1 so the build failed.

**Comment by ltrung** [comment]
>This fixes a bug where the VideoPreview component would not stop the media stream on the video preview video element because a ref was being unset before the unmount method could use the value of the ref.

Can we move this under Description of changes?

File: src/components/sdk/PreviewVideo/index.tsx (hunk 1 above)
Comment (michhyun1) [reply src/components/sdk/PreviewVideo/index.tsx:32]: yup,

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Add or remove event listener only when `el.current` reference is valid
   in `useFocusIn` and `useMouseMove` hooks.
+- Fix bug related to PreviewVideo component not releasing media stream when unmounted
```
Comment (devalevenkatesh) [inline CHANGELOG.md:15]: Nit: End with a ".".
Can we change PreviewVideo -> `PreviewVideo` ?

**Review by devalevenkatesh — COMMENTED** [review]
Fix looks good to me.

**Comment by michhyun1** [comment]
> You may want to update to Node 16 with NPM 7. It seems you changes package-lock.json to v1 so the build failed.

done. thanks

**Review by ltrung — APPROVED** [review]

**Pull request merged**